File: Root of Quadratic Equation/Root of Quadratic Equation.py
import math

print("Assuming Quadratic Equation of for ax^2+bx+c")
a,b,c = map(float,input("Please enter a,b and c ==> ").split())

d = (b**2) - (4*a*c)

if d<0:
    print("Roots are Imaginary")
elif d==0:
    r = -b / (2*a)
    print("Roots of Quadratic Equation are ==> ",r)
else:
    r1 = ((-b) + math.sqrt(d)) / (2*a)
    r2 = ((-b) - math.sqrt(d)) / (2*a)
    print("Roots of Quadratic Equation are ==> ",r1,r2)

File: Root of Quadratic Equation/Root of Quadratic Equation.cpp
#include<iostream>
#include<math.h>
using namespace std;

int main(){
    float a,b,c;
    cout<<"Assuming Quadratic Equation in the form of ax^2+bx+c";
    cout<<"Please enter a,b and c ==> ";
    cin>>a>>c>>c;
    float d = pow(b,2) - (4*(a*c));
    if(d<0){
        cout<<"Root is Imaginary";
    }else if(d==0){
        float r;
        r = -b / (2*a);
        cout<<"Root of Quadratic Equation is ==> "<<r;

    }else{
        float r1,r2;
        r1 = ((-b) + sqrt(d)) / (2*a);
        r2 = ((-b) - sqrt(d)) / (2*a);
        cout<<"Root of Quadratic Equation is ==> "<<r1<<" and "<<r2;
    }
}
